js/bitget.js

```diff
@@ -1111,6 +1122,293 @@ module.exports = class bitget extends Exchange {
+        //              "type": "deposit",
+        //              "amount": "19.44800000",
+        //              "status": "success",
+        //              "toAddress": "TRo4JMfZ1XYHUgnLsUMfDEf8MWzcWaf8uh",
+        //              "fee": null,
+        //              "chain": "TRC20",
+        //              "confirm": null,
+        //              "cTime": "1656407912259",
+        //              "uTime": "1656407940148"
+        //          }]
+        //      }
+        //
+        const rawTransactions = this.safeValue (response, 'data', []);
+        return this.parseTransactions (rawTransactions, currency, since, limit);
+    }
+
+    parseTransaction (transaction, currency = undefined) {
+        //
+        //     {
+        //         "id": "925607360021839872",
+        //         "txId": "f73a4ac034da06b729f49676ca8801f406a093cf90c69b16e5a1cc9080df4ccb",
+        //         "coin": "USDT",
+        //         "type": "deposit",
+        //         "amount": "19.44800000",
+        //         "status": "success",
+        //         "toAddress": "TRo4JMfZ1XYHUgnLsUMfDEf8MWzcWaf8uh",
+        //         "fee": null,
+        //         "chain": "TRC20",
+        //         "confirm": null,
+        //         "cTime": "1656407912259",
+        //         "uTime": "1656407940148"
+        //     }
+        //
+        const timestamp = this.safeString (transaction, 'cTime');
+        const networkId = this.safeString (transaction, 'chain');
+        const currencyId = this.safeString (transaction, 'coin');
+        const status = this.safeString (transaction, 'status');
+        return {
+            'id': this.safeString (transaction, 'id'),
+            'info': transaction,
+            'txid': this.safeString (transaction, 'txId'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'network': networkId,
+            'addressFrom': undefined,
+            'address': this.safeString (transaction, 'toAddress'),
+            'addressTo': this.safeString (transaction, 'toAddress'),
+            'amount': this.safeNumber (transaction, 'amount'),
+            'type': this.safeString (transaction, 'type'),
+            'currency': this.safeCurrencyCode (currencyId),
+            'status': this.parseTransactionStatus (status),
+            'updated': this.safeNumber (transaction, 'uTime'),
+            'tagFrom': undefined,
+            'tag': undefined,
+            'tagTo': undefined,
+            'comment': undefined,
+            'fee': undefined,
+        };
+    }
+
+    parseTransactionStatus (status) {
+        const statuses = {
+            'success': 'ok',
+            'Pending': 'pending',
+            'pending_review': 'pending',
+            'pending_review_fail': 'failed',
+            'reject': 'failed',
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    async fetchDepositAddress (code, params = {}) {
```

Comment:
Also, I've noticed that this method returns undefined if the address does not exist yet, bitget does not offer a "createDepositAddress" right?